src/main/frontend/components/onboarding/setups.cljs

```diff
@@ -46,12 +46,12 @@
    (cond
      (mobile-util/native-android?)
      [:div.px-4
-      "You can save them in your local storage, and use Logseq Sync or any third-party sync service to keep your notes sync with other devices. "
-      "If you prefer to use Dropbox to sync your notes, you can use "
```

Comment:
sync-desc-* keys can't rely on word ordering so this needs to be only one entry with the Dropsync entry as an argument.  Since there is formatted text involved, you'll probably need to provide a function translation like I [did here](https://github.com/logseq/logseq/pull/9716/commits/8be9b97a4761719eb67d7e85cc5c90ccb038cd38). If this is too difficult I'd just leave this sentence alone for now